syntax = "proto3";

package todo;

service TodoService {
  rpc CreateTodo (CreateTodoRequest) returns (CreateTodoResponse);
  rpc GetTodos (GetTodosRequest) returns (GetTodosResponse);
}

message CreateTodoRequest {
  string title = 1;
  string description = 2;
}

message CreateTodoResponse {
  int32 id = 1;
}

message GetTodosRequest {}

message GetTodosResponse {
  repeated Todo todos = 1;
}

message Todo {
  int32 id = 1;
  string title = 2;
  string description = 3;
}